.circleci/config.yml

```diff
@@ -32,7 +32,6 @@ executors:
   windows-installer:
     docker:
       - image: grafana/wix-toolset-ci:v3
-
```

Comment:
Why remove this blank line, since we use blank lines otherwise to separate top-level keys? Personally I wouldn't mind dropping these blank lines, but we should be consistent :)

Reply:
not sure, maybe prettier formatting that did it, it seems to have done a lot of indentation and quotation fixes